### ISSUE TITLE
Version 2.1 [includes information for packagers]

### ISSUE THREAD

**Issue by maandree** [erkin/ponysay#73]
Before we can tag 2.1 there are some minor things that we should do:

Confirm Spanish manpage: commit: https://github.com/erkin/ponysay/commit/972c1744a256ef1baa9c605fb27da95ac2d2dc5a

Make sure no up to date system identifies the Python script as Python 2. I edited the shebang to `python` instead of `python3` because emacs did not recognise `python3` withput `.py`, however both are recognised as Python 3 in Arch.
This should be check for Gentoo and Ubuntu, as any other system if some is using
something else.

Info manual should be updated to describe the pony files and how ponyquotes works .[I'll take that one]

**Packagers** for GNU/Linux distribution binary package repositories should make sure /var/cache/ponysay is either emptied or complete removed before all files are removed. This is to make sure no bad kmsponies are lurking around when there format is updated, this can happen for this version and any future version.

In other words, a pre-installation hook is needed:

```
if [-d "/var/cache/ponysay" ]; then
    rm -r "/var/cache/ponysay"`
fi
```
